README.md

```diff
@@ -285,6 +286,15 @@ Let's assume that you have a 16x16 pixels image and a link to the service which
 </div>
 ```
 
+### Light / dark theme
+It's possible to use alternative (dark-mode suitable) styling by adding `likely-light` class to the main `div.likely`
+```html
+<div class="likely likely-light">
+    <!-- List of services -->
+</div>
+```
+Additionally, if your website is responsive to users' color theme preferences, having `.likely-color-theme-based` instead of `.likely-light` will result in having regular styling for light-mode users and `.likely-light` styling for dark-mode users.
```

Comment:
I'd remove the last part of the sentence because it's already said in the beginning of the section.
"`.likely-light` styling for dark-mode users."

Reply:
Thinking of it, let's also add an alias for .likely-dark, that would mean the same.

Reply:
https://github.com/NikolayRys/Likely/pull/196/commits/0bcc3831ac32f9162f5268c2f2359b5421279bae

> Thinking of it, let's also add an alias for .likely-dark, that would mean the same.